<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesEquipment</class>
 <widget class="QWidget" name="PreferencesEquipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QLabel" name="label_cylinders">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>CYLINDERS</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Default cylinder in the Cylinders table of the Equipment tab</string>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>

      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Select a default cylinder</string>
        </property>
       </widget>
      </item>

      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="default_cylinder"/>
        </item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="include_unused_tanks">
     <property name="text">
      <string>Show unused cylinders in the Cylinders table of the Equipment tab and include them in export / copy / merge</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="display_default_tank_infos">
     <property name="text">
      <string>Show default cylinder sizes in the cylinders selection of the equipment tab</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>


  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
